### PR TITLE
CC-3201 "Allow issuing start/stop/restart commands to multiple services via API"

### PR DESCRIPTION
**Pull request by johnlhamilton**
New REST endpoints:
PUT services/restartServices
PUT services/startServices
PUT services/stopServices

NEW CLI options
serviced service start [SERVICE] [ANOTHER SERVICE] ...
serviced service stop [SERVICE] [ANOTHER SERVICE] ...
serviced service restart [SERVICE/INSTANCE] [ANOTHER SERVICE/INSTANCE]...

### CODE REVIEW

File: web/resources.go [hunk 1]
```diff
@@ -581,6 +581,33 @@ func restRestartService(w *rest.ResponseWriter, r *rest.Request, client *daoclie
 	w.WriteJson(&simpleResponse{"Restarted service", serviceLinks(serviceID)})
 }
 
+// restRetartServices starts the services with the given ids and all of their children
+func restRestartServices(w *rest.ResponseWriter, r *rest.Request, client *daoclient.ControlClient) {
+
+	var serviceRequest dao.ScheduleServiceRequest
+	err := r.DecodeJsonPayload(&serviceRequest)
+	if err != nil {
+		glog.V(1).Info("Could not decode service payload: ", err)
```
Comment (spopezen) [inline web/resources.go]: No glog!

File: web/resources.go [hunk 2]
```diff
@@ -622,6 +649,33 @@ func restStartService(w *rest.ResponseWriter, r *rest.Request, client *daoclient
 	w.WriteJson(&simpleResponse{"Started service", serviceLinks(serviceID)})
 }
 
+// restStartServices starts the services with the given ids and all of their children
+func restStartServices(w *rest.ResponseWriter, r *rest.Request, client *daoclient.ControlClient) {
+
+	var serviceRequest dao.ScheduleServiceRequest
+	err := r.DecodeJsonPayload(&serviceRequest)
+	if err != nil {
+		glog.V(1).Info("Could not decode service payload: ", err)
```
Comment (spopezen) [inline web/resources.go]: Same.

File: web/resources.go [hunk 3]
```diff
@@ -641,14 +695,37 @@ func restStopService(w *rest.ResponseWriter, r *rest.Request, client *daoclient.
...
 	w.WriteJson(&simpleResponse{"Stopped service", serviceLinks(serviceID)})
 }
 
+// restStopServices stops the services with the given ids and all of their children
+func restStopServices(w *rest.ResponseWriter, r *rest.Request, client *daoclient.ControlClient) {
+
+	var serviceRequest dao.ScheduleServiceRequest
+	err := r.DecodeJsonPayload(&serviceRequest)
+	if err != nil {
+		glog.V(1).Info("Could not decode service payload: ", err)
```
Comment (spopezen) [inline web/resources.go]: You know what to do.

File: web/resources.go (hunk 1 above)
Comment (johnlhamilton) [inline web/resources.go]: Nooooooooooooooo

**Pull request merged**